src/shapes/object.class.js

```diff
@@ -1905,9 +1907,13 @@
     }
   });
 
-  fabric.util.createAccessors && fabric.util.createAccessors(fabric.Object);
+  // fabric.util.createAccessors && fabric.util.createAccessors(fabric.Object);
 
-  extend(fabric.Object.prototype, fabric.Observable);
+  fabric.Object.prototype.on = Observable.prototype.on;
+  fabric.Object.prototype.once = Observable.prototype.once;
+  fabric.Object.prototype.off = Observable.prototype.off;
+  fabric.Object.prototype.fire = Observable.prototype.fire;
+  fabric.Object.prototype._removeEventListener = Observable.prototype._removeEventListener;
```

Comment:
Why not Object.assign(fabric.Object.prototype, fabric.Observable) ?

Reply:
I tried that but it doesn't work

Reply:
because methods aren't enumerable on class proto